package.json

```diff
@@ -69,7 +70,8 @@
     "sinon-sandbox": "^1.0.2"
   },
   "peerDependencies": {
-    "react": ">=0.14"
+    "react": ">=0.14",
+    "prop-types": ">=15.5.6"
```

Comment:
No, if they require different versions, they need multiple copies; if not, `npm` will dedupe them.